akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -1779,6 +1779,8 @@ final class Source[+Out, +Mat](delegate: scaladsl.Source[Out, Mat]) extends Grap
   def watchTermination[M]()(matF: function.Function2[Mat, CompletionStage[Done], M]): javadsl.Source[Out, M] =
     new Source(delegate.watchTermination()((left, right) ⇒ matF(left, right.toJava)))
 
+  def monitor: javadsl.Source[Out, Pair[Mat, FlowMonitor[Out]]] = viaMat(Flow.monitor[Out](), Keep.both[Mat, FlowMonitor[Out]])
```

Comment:
"Only on Flow" means, not here either. `Source.via(Flow)` = `Source` so no need to add this.